.github/workflows/build-suite-native-preview.yml

```diff
@@ -0,0 +1,59 @@
+name: "[Build] suite-native preview"
+
+on:
+  push:
+    # push develop(default) branch is necessary for this action to update its fingerprint database
+    branches: [develop]
+  pull_request:
+    paths:
+      - "suite-native/**"
+      - "packages/react-native-usb/**"
+      - "packages/transport-native/**"
+      - "yarn.lock"
+      # list of paths is not complete, but it's always possible to dispatch manually
```

Comment:
I would suggest add suite-common/** so people from other teams could easily test changes that can affect mobile app

Reply:
Good point. I'll added, but maybe in next PR. I found out that currently it does the build on every new push even if the runtimeVersion is not changed, so I have to fix that first.